scripts/src/java/org/oppia/android/scripts/proto/coverage.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package proto;
+
+option java_package = "org.oppia.android.scripts.proto";
+option java_multiple_files = true;
+
+message CoverageReport {
```

Comment:
Reminder to add documentation comments for all protos and all fields in this file.

Reply:
Added Docs @BenHenning. But I have trouble understanding the difference between the 
  
Block number and branch number in the coverage data.
  - int32 **block_number** = 2;
  - int32 **branch_number** = 3;
  
These are generally represented as: 
```
BRDA:46,0,0,1
BRDA:46,0,1,1
```
what does a block and a branch mean?
So I assumed branch would be like 
```
if(condition) { // branch 0
// code block
} else { // branch 1
// code block
}
```
if that's right then what is a block in it?
```
if(condition) { // is the condition called a block - condition 1 - block 0, condition 2 - block 1
// code block // or the actual code block a block?
} else { // branch 1
// code block
}
```
I am also looking into few results to figure this out, but if it would be possible to share some insights on this that would be helpful.

Reply:
I'm not 100% certain that I understand the distinction, either. I'm having some difficulty in finding a well-define page documenting the file format to explain the distinction, as well.

I suspect we can probably ignore all but lines for coverage since that's what we're presumably using for computing line-of-code percentage. Branch & block analysis could be interesting to add in the future to try and enforce more thorough tests (or for the final HTML/MD report generation).

Reply:
Got it, Ben. As discussed, now the current implementations will only represent the line coverage for both HTML and MD reports.